test/unit/lib/static/components/state/state-error.jsx

```diff
@@ -101,4 +101,62 @@ describe('<StateError/> component', () => {
             assert.equal(component.find('.details__content .foo-bar').text(), ['some-hint']);
         });
     });
+
+    describe('error snippet', () => {
+        it('should be rendered in preformatted tags', () => {
+            const error = {snippet: `\x1B[90m   . | // /some-file-path.js\x1B[39m")}
+                . |
+                9 | some line
+               10 | other line
+            `};
+
+            const component = mkStateErrorComponent({result: {error}});
+            component.find('.details__summary').last().simulate('click');
+
+            assert.isTrue(component.find('.details__content').html().startsWith('<div class="details__content"><pre>'));
```

Comment:
For some reason, we cant use `.find` to get into this `<pre>` node. I think it is because i used `dangerouslySetInnerHTML`